App/Sagas/TextileEventsSagas.ts

```diff
@@ -196,15 +197,21 @@ function* handleNodeEvents() {
 
 function* initializeTextile() {
   try {
-    const verbose = yield select(PreferencesSelectors.verboseUi)
-    const phrase: string | undefined = yield call(
-      Textile.initialize,
-      verbose,
-      true
+    const initialized = yield call(
+      Textile.isInitialized,
+      AppConfig.textileRepoPath
     )
-    if (phrase) {
+    const verbose = yield select(PreferencesSelectors.verboseUi)
+    if (!initialized) {
+      const phrase = yield call(
+        Textile.initializeCreatingNewWalletAndAccount,
+        AppConfig.textileRepoPath,
+        verbose,
+        true
+      )
       yield put(accountActions.setRecoveryPhrase(phrase))
     }
+    yield call(Textile.launch, AppConfig.textileRepoPath, verbose)
```

Comment:
Above is an example of using the new init api to replicate the functionality that existed before (just always create a new account). This will be changed with a fork in the logic depending on if the user chooses to create a new account or sync an existing account.

Reply:
1. Always first check if Textile is already initialized
2. If not, initialize Textile, either creating a new wallet and account or by specifying an existing account seed.
3. Always launch Textile